#!/usr/bin/python2
PACKAGE = "cooperative_driving_application"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

# General behavior parameters
gen.add("beaconing_interval",         int_t, 0, "The beaconing interval for CAM messages",            1,  1, 20)
gen.add("enable_beaconing",           bool_t, 0, "Enable or disable beaconing",  True)
gen.add("announce",                   bool_t, 0, "Send command",  False)

exit(gen.generate(PACKAGE, "application", "applicationServer"))

